code/game/objects/items/devices/radio/radio.dm

```diff
@@ -474,6 +475,13 @@ var/global/list/default_medbay_channels = list(
 		// we're done here.
 		return 1
 
+//VOREStation Add Start
+	if(bluespace_radio)
+		return Broadcast_Message(connection, M, voicemask, pick(M.speak_emote),
+					  src, message, displayname, jobname, real_name, M.voice_name,
+					  0, signal.data["compression"], list(0), connection.frequency,verb,speaking)

```

Comment:
Missing a few spaces after commas in the last few parameters here

Reply:
Yeah, it is the same in the other instances of Broadcast_Message in radio.dm, not sure why that is. I kept it for the sake of consistency.

Reply:
It seems like this should be below the !connection test below, given that you're still using connection. Actually do you know WHY this makes it send even though there's no relay? Is it just the list(0) for the zlevel?

Reply:
Yes, it is just the list(0) for the zlevel.